vm/devices/pci/pci_core/src/cfg_space_emu.rs

```diff
@@ -478,30 +478,19 @@ impl ConfigSpaceType0Emulator {
 
         match HeaderType00(offset) {
             HeaderType00::STATUS_COMMAND => {
-                let command = match cfg_space::Command::from_bits(val as u16) {
-                    Some(command) => command,
-                    None => {
-                        tracelimit::warn_ratelimited!(offset, val, "setting invalid command bits");
-                        // still do our best
-                        cfg_space::Command::from_bits_truncate(val as u16)
-                    }
+                let mut command = cfg_space::Command::from_bits(val as u16);
+                if command.reserved() != 0 || command.reserved2() != 0 {
```

Comment:
This is a dangerous way to do this check, because reserved bits might become unreserved.

Instead, I'd do a positive check like:
`let supported = cfg_space::Command::with_foo(true).with_bar(0x3).with_baz(true).into_bits()` and then check against that mask.